src/OrchardCore.Modules/OrchardCore.Workflows/Views/Items/HttpRequestTask.Fields.Edit.cshtml

```diff
@@ -55,6 +55,16 @@
     <span class="hint">@T["A comma-separated list of HTTP response status codes to handle as activity outcomes. For example: 200, 404, 500"]</span>
 </div>
 
+<div class="mb-3 form-group">
+    <div class="custom-control custom-checkbox">
+        <input type="checkbox" class="custom-control-input" asp-for="IsBinary" checked="@Model.IsBinary" />
+        <label class="custom-control-label" asp-for="IsBinary">@T["Is Binary"]</label>
+        <span class="hint dashed">
+            @T["Is Binary"]
```

Comment:
If this remains (but reading a bit your convo I'm not sure), then add an explanation of why this option is useful and when would you use it. Also for `HttpResponseTask`

Reply:
There are many use cases:
- creating a zip file in your workflow script and return it,
- creating an image in your workflow script and return it,
- accessing an external API from your workflow script which results binary response.

If this PR doesn't have any value for others, let's close it. I've just shared a working solution that handles a few use cases I've encountered in the past.

Reply:
I'm not arguing against the usefuleness of being able to send a binary response. By "this" I meant the commented "Is Binary" checkbox. My comment is about if this checkbox remains in its current form, as opposed to, e.g., having a dropdown with more options, or some auto-detection.